internal/uvm/scsi/backend.go

```diff
@@ -197,14 +197,16 @@ func mountRequest(controller, lun uint, path string, config *mountConfig, osType
 		}
```

Comment:
There is an error check on line 192 that should have the new fields added to it. Looks like I forgot to update it for Partition as well.

Reply:
I think the line numbers got messed up here. You're referring to the check for wcow right?

Reply:
Updated